### PR TITLE
Update config-example.yaml

### PR DESCRIPTION
**Pull request by rayzhou2017**

### CODE REVIEW

File: docs/config-example.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ spec:
     address: ""
     port: "6443"
   kubernetes:
-    version: v1.17.4
+    version: v1.17.5
     imageRepo: kubekey
```
Comment (rayzhou2017) [inline docs/config-example.md:26]: What does imageRepo mean here? Does it mean the K8s images are from hub.docker.com/kubekey?

Comment (pixiake) [reply docs/config-example.md:26]: Yes，I haven't updated here. `hub.docker.com/kubesphere` is default.

**Pull request merged**